stats/s-bandwidth

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+LKP_SRC = ENV['LKP_SRC'] || File.dirname(File.dirname(File.realpath($PROGRAM_NAME)))
+
+require "#{LKP_SRC}/lib/statistics"
+
+workload = 0
+data = false
+pattern = /^\S*\s+(\d*\.?\d+)\s+(\d*\.?\d+)\s+(\d*\.?\d+)\s+(\d*\.?\d+)\s+(\d*\.?\d+)\s+(\d*\.?\d+)\s+(\d*\.?\d+)\s+/
+while (line = STDIN.gets)
+  if [data]
+    if line.match(pattern)
+      puts 'tps: ' + $1
+      puts 'MB_read/s: ' + $2
+      puts 'MB_wrtn/s: ' + $3
+      puts 'MB_dscd/s: ' + $4
+      puts 'MB_read: ' + $5
+      puts 'MB_wrtn: ' + $6
+      puts 'MB_dscd: ' + $7
+    end
+    data = false
+  else
+    if line.match(/^Device.*$/)
```

Comment:
Is it possible that the pattern is expanded to include "Device", so the code can be below to remove the usage of data variable.
```
while (line = STDIN.gets)
   next unless line.match(pattern)
   puts 'tps: ' +$1
end
```

Reply:
I kept the variable, that helps me to match the line containing the values, that is written right after each line that starts with "Device".
I need to filter with the previous line since the name of the hard drive may vary, and it's not a reliable way of filtering.